test/suites/nodeclaim/vm_extension_test.go

```diff
@@ -0,0 +1,38 @@
+package nodeclaim_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/samber/lo"
+
+	"sigs.k8s.io/karpenter/pkg/test"
+)
+
+var _ = Describe("VMExtension", func() {
```

Comment:
Just a note to myself to include test section for AKSIdentifyingExtension

Reply:
AKSIdentifyingExtension == billing extension